TX/Generic LR1121 Gemini.json

```diff
@@ -0,0 +1,40 @@
+{
+    "serial_rx": 1,
+    "serial_tx": 1,
+
+    "radio_miso": 33,
+    "radio_mosi": 32,
+    "radio_sck": 25,
+
+    "radio_busy": 36,
+    "radio_dio1": 37,
+    "radio_nss": 27,
+    "radio_rst": 26,
+
+    "radio_busy_2": 39,
+    "radio_dio1_2": 34,
+    "radio_nss_2": 13,
+    "radio_rst_2": 21,
+
+    "power_min": 0,
+    "power_high": 3,
+    "power_max": 3,
+    "power_default": 0,
+    "power_control": 0,
+    "power_values": [12,16,19,22],
+    "power_values_dual": [-10,-6,-3,1],
+
+    "led_rgb": 22,
+    "led_rgb_isgrb": true,
+    "ledidx_rgb_status": [0],
+    "ledidx_rgb_boot": [0],
+
+    "radio_dcdc": true,
+
+    "use_backpack": false,
+    "debug_backpack_baud": 460800,
+    "debug_backpack_rx": 3,
+    "debug_backpack_tx": 1,
```

Comment:
Probably don't want to use the main port for backpack, especially as you've used it for the s.port above.

Reply:
Fixed.

Yep, testing left overs.